cmux-tui/docs/PR-INTENT-BOARD.md

```
@@ -1,6 +1,75 @@
 # cmux TUI PR intent and merge board
 
-## Current reconciliation: main `af31628f7b0b2f6c34e184049254fa2fe91f285d`
+## Current reconciliation: main `f8a66a902eb2d7fe36010dedad1dd6a6ce64f25c`
+
+Updated main is `d0b3b737a26f6afa6565b6c0160a31700abe6e21`, after merged #11012.
+Live direct-TUI open rows: #10891, #10966, #10969, #10990, #10992, #10993,
+#10994, #10995, #10996, #11000, #11002, #11013, #11017, and #11019. Exact heads
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)